MediaGallerySynchronization/Model/SynchronizeFiles.php

```diff
@@ -52,25 +52,33 @@ public function __construct(
      */
     public function execute(array $files): void
     {
-        $failedFiles = [];
-        foreach ($files as $file) {
-            try {
-                $this->saveAsset->execute([$this->createAssetFromFile->execute($file)]);
-            } catch (\Exception $exception) {
-                $this->log->critical($exception);
-                $failedFiles[] = $file->getFilename();
-            }
-        }
-
-        if (!empty($failedFiles)) {
+        try {
+            $this->saveAsset->execute($this->createAsstesFromFiles($files));
+        } catch (\Exception $exception) {
+            $this->log->critical($exception);
             throw new LocalizedException(
                 __(
                     'Could not update media assets for files: %files',
                     [
-                        'files' => implode(', ', $failedFiles)
+                        'files' => implode(', ', $files)
```

Comment:
I don't think outputing the files list makes sense with batches, lets not include the files list in the exception message

Reply:
ok